lib/filter_cli.c

```diff
@@ -68,14 +68,19 @@ static int acl_get_seq_cb(const struct lyd_node *dnode, void *arg)
  * \param[in] xpath the XPath to look for the sequence leaf.
  * \returns next unused sequence number.
  */
-static long acl_get_seq(struct vty *vty, const char *xpath)
+static int64_t acl_get_seq(struct vty *vty, const char *xpath, bool is_remove)
```

Comment:
two comments here:
1. please update the function documentation comment (e.g. returns `-1` if sequence number wrapped)
2. I think the `is_remove` logic should be moved to `acl_remove_if_empty` since its the only place benefiting from this. this will save you from having to modify all other callers.

Reply:
1. Done
2. The logic is mainly for **non-remove** cases, so as to give a warning to users. So, it can't be moved into `acl_remove_if_empty`/`plist_remove_if_empty()`.